environments/dev/yii_test

```diff
@@ -24,5 +26,10 @@ $config = yii\helpers\ArrayHelper::merge(
 );
 
 $application = new yii\console\Application($config);
-$exitCode = $application->run();
+try {
+    $exitCode = $application->run();
+} catch (\Exception $exception) {
```

Comment:
That would cause error handler to never fire preventing logging exceptions and errors. Also that would mess up formatting.

I thin what you want to change is https://github.com/yiisoft/yii2/blob/master/framework/base/ErrorHandler.php#L112 but that can't be done since it would affect testing (tests would exit after first failure without suite completion).